test/test-execute/exec-systemcallfilter-with-errno-name2.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Test for SystemCallErrorNumber
+
+[Service]
+ExecStart=/bin/sh -x -c 'test "$$(uname -a 2>&1)" = "uname: cannot get system name: Invalid or incomplete multibyte or wide character"'
```

Comment:
Hmm, hardcoding the error message... not nice. Maybe something like this:
```ini
ExecStart=/bin/python3 -c 'import os\ntry: os.uname()\nexcept Exception as e: exit(e.errno)'
```
?

Reply:
OK. Converted to a python script.